### PR TITLE
removed physics and added floor collisions

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

**Comment by floweisshardt** [comment]
@abubeck you introduces this physics parameters to accelerate simulation speed. But this caused problems that the wheels felt into the floor and the robot didn't move any more. Do you have an update here?
If not, I'm going to accept this PR to fix it.

**Comment by fmessmer** [comment]
Looks good, now all floors are actually at z=0.0!
There is just a minor overlap between floor_kitchen and floor_student_area causing a graphical glitch (the rendering switches between carpet and white from time to time)

Also, please see my inline remark...

File: cob_gazebo_worlds/worlds/empty_apartment.world [hunk 1]
```diff
@@ -71,19 +52,36 @@
...
+            </plane>
+          </geometry>
+          <surface>
+            <friction>
+              <ode>
+                <mu>100</mu>
+                <mu2>50</mu2>
+              </ode>
+            </friction>
+          </surface>
```
Comment (fmessmer) [inline cob_gazebo_worlds/worlds/empty_apartment.world:69]: where did you get the <surface>...</surface> tag from?

from the gazebo sdf format:

```
<ode>
Description: ODE friction parameters
Required: 0    Type: n/a
    <mu>
    Description: Coefficient of friction in the range of [0..1].
    Required: 0    Type: double    Default: 1
    <mu2>
    Description: Second coefficient of friction in the range of [0..1]
    Required: 0    Type: double    Default: 1
    <fdir1>
    Description: 3-tuple specifying direction of mu1 in the collision local reference frame.
    Required: 0    Type: vector3    Default: 0 0 0
    <slip1>
    Description: Force dependent slip direction 1 in collision local frame, between the range of [0..1].
    Required: 0    Type: double    Default: 0
    <slip2>
    Description: Force dependent slip direction 2 in collision local frame, between the range of [0..1].
    Required: 0    Type: double    Default: 0
```

thus, a value of 100 (50) does not seem right
what's more, it seems gazebo does not simulate friction...

can you also drive around without the <surface>...</surface> part? if yes, we should remove this for each floor_*

**Comment by NadiaHG** [comment]
I got the parameters from ground_plane (https://github.com/gazebosim/gazebo/blob/master/sdf/models/ground_plane.model), the default plane that uses empty.world (https://bitbucket.org/osrf/simulator_gazebo/src/13e6ffd3ddb0ff2cee445ab355f522ae223373d5/gazebo_worlds/worlds/empty.world?at=default)

**Pull request merged**